modules/components/src/Admin/ProjectsTable.js

```diff
@@ -25,7 +26,14 @@ export default ({
     allowTSVExport={false}
     customTypes={{
       entity: props => {
-        return <Link to={`/projects/${props.value}`}>{props.value}</Link>;
+        return (
+          <Link
+            onClick={() => setActiveProject({ activeProject: props.value })}
+            to={`/projects/${props.value}`}
```

Comment:
@alex-wilmer - I thought there should be some better way of tying a route change to state, but wasn't sure, is this how you'd do it?

Reply:
You can derive the state from the url so you don't have to maintain it.
That will likely require a bit of refactoring of the routes which is probably out of scope for this ticket.

Reply:
yeah I had `activeProject` before I added router at all. we can eventually just use route params for that, agreed that its out of scope here